### PR TITLE
Release PRs Sync 

### PR DESCRIPTION
**Pull request by pavjacko**
## Description

- Schema fixes for `rnv new`
- schema fixes for bootstrap questions
- support for dynamic engine configurations 
- support for engine specific template overrides
- refactor c into getContext

## Related issues

Included PRs to avoid larger conflicts due to API signature changes and streamlined QA:

- https://github.com/flexn-io/renative/pull/1451
- https://github.com/flexn-io/renative/pull/1438 (fixed in this PR)
- https://github.com/flexn-io/renative/pull/1447 (additional fixes in this PR)
- https://github.com/flexn-io/renative/pull/1453
- https://github.com/flexn-io/renative/pull/1439
- https://github.com/flexn-io/renative/pull/1450 (additional fixes in this PR)

- GH issues

## Npm releases

n/a

### CODE REVIEW

File: packages/app-harness/src/app/index.tsx [hunk 1]
```diff
@@ -58,8 +82,8 @@ const App = () => {
                     <TestCase id={3} title="Orientation support ">
                         <OrientationLocker
                             orientation={PORTRAIT}
-                            onChange={(orientation) => console.log('onChange', orientation)}
-                            onDeviceChange={(orientation) => console.log('onDeviceChange', orientation)}
+                            onChange={(orientation) => logDebug(`onChange ${orientation}`)}
```
Comment (mihaiblaga89) [inline packages/app-harness/src/app/index.tsx:85]: wouldn't it be better if we refactor logDebug to use params?

Comment (pavjacko) [reply packages/app-harness/src/app/index.tsx:85]: 👌  done

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
can't run tizen and tizenwatch on app-harness and template-starter on sim
```
pauliusguzas@MBP-de-Flexn app-harness % npx rnv run -p tizen -r

┌─────────────────────────────────────────────────────────────────┐
│ ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗ │
│ ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝ │
│ ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗   │
│ ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝   │
│ ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗ │
│ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝ │
│ 🚀 v:1.0.0-rc.12 | renative.org | 3/12/2024, 12:49:11 PM        │
│ $ npx rnv run -p tizen -r                                       │
└─────────────────────────────────────────────────────────────────┘
info: Load engine: @rnv/engine-rn (./node_modules/@rnv/engine-rn)
info: Load engine: @rnv/engine-rn-tvos (./node_modules/@rnv/engine-rn-tvos)
info: Load engine: @rnv/engine-rn-next (./node_modules/@rnv/engine-rn-next)
info: Load engine: @rnv/engine-rn-web (./node_modules/@rnv/engine-rn-web)
info: Load engine: @rnv/engine-rn-electron (./node_modules/@rnv/engine-rn-electron)
info: Current engine: engine-rn-web (../engine-rn-web)
task: ○ run [1]
info: Build hooks not complied. BUILDING...
task: ○ configure [1]
task: ○ platform configure [1]
task: ○ project configure [1]
task: ○ workspace configure [1]
task: ✔ workspace configure [1]
task: ○ install [1]
task: ✔ install [1]
task: ○ install [2]
task: ✔ install [2]
task: ○ app configure [1]
info: Current app config: harness
task: ✔ app configure [1]
info: Plugin overrides disabled for: react-native-gesture-handler with disablePluginTemplateOverrides. SKIPPING
info: ../../node_modules/@react-native-community/cli-platform-ios/build/tools/listIOSDevices.js overriden by: ../rnv/pluginTemplates/@react-native-community/cli-platform-ios/overrides@11.3.7.json
info: Plugin overrides disabled for: next with disablePluginTemplateOverrides. SKIPPING
info: Plugin overrides disabled for: react-native-permissions with disablePluginTemplateOverrides. SKIPPING
task: ✔ project configure [1]
info: Current Build Scheme: debug
task: ○ install [3]
task: ✔ install [3]
info: You passed -r argument. "./platformBuilds/harness_tizen" CLEANING...DONE
task: ✔ platform configure [1]
✔ Executing: /Users/pauliusguzas/tizen-studio/tools/ide/bin/tizen security-profiles list -n RNVanillaCert
info: Found custom assetSources at ../../node_modules/@flexn/assets-renative-outline/assets/tizen. Will be used to generate assets.
task: ✔ configure [1]
info: Your tizen devServer at port 8087 is not running. Starting it up for you...
info: Your tizen devServerHost 127.0.0.1 at port 8087 is not running. Starting it up for you...
┌─────────────────────────────────────────────────────────────────┐
│ BUNDLER STARTED | 3/12/2024, 12:49:14 PM | rnv@1.0.0-rc.12      │
│ $ npx rnv run -p tizen -r                                       │
├─────────────────────────────────────────────────────────────────┤
│ Project: @rnv/app-harness@1.0.0-rc.12                           │
│ Workspace: rnv                                                  │
│ Platform (-p): tizen                                            │
│ Engine: engine-rn-web                                           │
│ Template: @rnv/template-starter@1.0.0-rc.12                     │
│ App Config (-c): harness                                        │
│ Build Scheme (-s): debug                                        │
│ Target (-t): T-samsung-7.0-x86                                  │
│ Reset Project (-r): true                                        │
│ Supported Platforms: tizen                                      │
│ Env Info: darwin | x64 | node v18.17.0                          │
│ Executed Time: 0h:0m:2s:901ms                                   │
│ Project location: ./platformBuilds/harness_tizen                │
└─────────────────────────────────────────────────────────────────┘
warn:○ run: You are missing chii. You can install via npm i -g chii) Trying to use weinre next
warn:○ run: You are missing weinre. Skipping debug. install via npm i -g weinre
(node:39114) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:39114) [DEP_WEBPACK_DEV_SERVER_ON_BEFORE_SETUP_MIDDLEWARE] DeprecationWarning: 'onBeforeSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
Starting the development server...

Compiled successfully!

You can now view @rnv/app-harness in the browser.

  Local:            http://localhost:8087
  On Your Network:  http://192.168.8.181:8087

Note that the development build is not optimized.
To create a production build, use npm run build.

assets by path static/ 3.53 MiB
  asset static/js/bundle.js 3.22 MiB [emitted] (name: main) 1 related asset
  asset static/media/Inter-Light.58a136ec3193fa8a63e5.ttf 303 KiB [emitted] [immutable] [from: appConfigs/base/fonts/Inter-Light.ttf] (auxiliary name: main)
  asset static/media/logo.63d54a55b067ffc3e842.png 12.1 KiB [emitted] [immutable] [from: platformAssets/runtime/logo.png] (auxiliary name: main)
asset index.html 1.77 KiB [emitted]
asset asset-manifest.json 356 bytes [emitted]
cached modules 2.7 MiB (javascript) 303 KiB (asset) 28.8 KiB (runtime) [cached] 685 modules
./platformAssets/runtime/logo.png 42 bytes (javascript) 12.1 KiB (asset) [built]
webpack 5.90.1 compiled successfully in 2183 ms
Files successfully emitted, waiting for typecheck results...
Issues checking in progress...
✔ Executing: /Users/pauliusguzas/tizen-studio/tools/sdb devices
⠹ Launch command : "/Users/pauliusguzas/tizen-studio/platforms/tizen-7.0/tv-sams
✔ Executing: /Users/pauliusguzas/tizen-studio/tools/emulator/bin/em-cli launch --name T-samsung-7.0-x86
✔ Waiting for emulator to boot...
✖ FAILED: /Users/pauliusguzas/tizen-studio/tools/ide/bin/tizen build-web -- /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/build -out /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/intermediate
error: ⨯ ○ run: COMMAND:

/Users/pauliusguzas/tizen-studio/tools/ide/bin/tizen build-web -- /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/build -out /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/intermediate

FAILED with ERROR:

Error occurred during build!
	java.lang.Exception: The specified directory(/Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/build) is not vaild
Total time: 00:00:00.021

Error: Command failed with exit code 1: /Users/pauliusguzas/tizen-studio/tools/ide/bin/tizen build-web -- /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/build -out /Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/intermediate
Error occurred during build!
	java.lang.Exception: The specified directory(/Users/pauliusguzas/Desktop/renative/renative/packages/app-harness/platformBuilds/harness_tizen/build) is not vaild
Total time: 00:00:00.021
    at makeError (/Users/pauliusguzas/Desktop/renative/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/renative/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


```

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
none of the issues from https://github.com/flexn-io/renative/pull/1453 were fixed

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
kaios sim launch isn't fixed image

**Comment by pauliusguzas** [comment]
> kaios sim launch isn't fixed image

this is solved

**Pull request merged**